src/wp-includes/class-wp-metadata-lazyloader.php

```diff
@@ -133,15 +133,19 @@ public function reset_queue( $object_type ) {
 	 * @since 4.5.0
 	 *
 	 * @param mixed $check The `$check` param passed from the 'get_term_metadata' hook.
+	 * @param int   $object_id ID of the object metadata is for.
 	 * @return mixed In order not to short-circuit `get_metadata()`. Generally, this is `null`, but it could be
 	 *               another value if filtered by a plugin.
 	 */
-	public function lazyload_term_meta( $check ) {
+	public function lazyload_term_meta( $check, $object_id ) {
```

Comment:
This parameter needs to be optional, given that this function is public. Otherwise this is a backward compatibility break.

Reply:
Good feedback. Created a new method.